gnome/gnome-voice/Portfile

```diff
@@ -23,6 +25,8 @@ checksums           rmd160  7e45780cb9aa8105639f5a01a1d7fc368d7d43ee \
                     sha256  2c92a4acf3474c76758cead7469ed73b42f9352b9c949dd9febea36b05f39014 \
                     size    105728
 
+perl5.branches      5.34
```

Comment:
Only used by `depends_build`. Maybe this line could go?

Reply:
Doesn't appear to be telling the build system which perl to use.